src/modules/PublicLab.RichTextModule.Table.js

```diff
@@ -95,5 +97,16 @@ module.exports = function initTables(_module, wysiwyg) {
         $('.woofmark-command-table').popover('toggle');
       });
     });
+
+    $(':not(".woofmark-command-table")').click((e) => {
+      // check to see that the clicked element isn't fa-table icon, else when you click on the fa-table icon, the popover will close
+      if (popoverIsOpen && $('.woofmark-command-table').children()[0] != e.target) {
+        const popoverContainer = document.querySelector('.popover');
+        const isChildElement = popoverContainer.contains(e.target);
+        if (popoverIsOpen && !e.target.classList.contains("woofmark-command-table") && !isChildElement) {
+          $('.woofmark-command-table').click();
+        }
+      }
+    });
```

Comment:
Also, why is this code inside `.woofmark-command-insert` click function? I thought you were trying to catch all clicks that were made outside the button.

Reply:
I too was confused about it but @NARUDESIGNS and I agreed that, we will be checking for it only when popover is open.

Reply:
The idea is; we only want to start listening to any other click on the page to close the custom insert popover when the custom insert tool button has been clicked. So unless the custom insert tool button has been clicked, we do not want to listen to a random click on the page to try to close the popover (after all the popover hasn't been displayed since the custom insert isn't clicked yet).
But when we just listen to clicks on the outer part of the page even when no popover is open, Chrome console throws an error. Those popovers are created dynamically by bootstrap and so unless they're open, we do not want to listen to other clicks to close them.

This was the idea. @TildaDares

Reply:
> Also, why is this code inside `.woofmark-command-insert` click function? I thought you were trying to catch all clicks that were made outside the button.

Also, I think you meant `.woofmark-command-table` click function. @TildaDares

Reply:
That makes a lot of sense @NARUDESIGNS. Thanks!

@RaviAnand111 I think your solution works fine. There’s no need to change anything.